pkg/kubernetes/views/topology-page.html

```diff
@@ -46,7 +46,7 @@ <h3 translatable="yes">Node</h3>
             <button id="delete-entity" title="Delete" class="btn btn-danger btn-delete pficon pficon-delete delete-entity" data-toggle="modal" data-target="#delete-entity-dialog" data-id="{{id}}" data-key="{{selected.key}}"></button>
             <button id="adjust-entity-btn" title="Adjust"  class="btn btn-default pficon pficon-edit adjust-entity"  data-toggle="modal" data-target="#adjust-entity-dialog"  data-id="{{id}}" data-key="{{selected.key}}"></button>
           </div>
-          <div ng-if='selected.kind === "Service"'>
+          <div ng-if='selected.kind === "Service" || selected.kind === "Route"'>
```

Comment:
Shouldn't any resource be deletable? Why are we special casing them all? I understand that Pod has a separate message, but we should just have one fallback delete button for all other topology resources, no?